[FIX] split MEG files should be listed separately in scans.tsv (#735)
bids-standard/bids-specification: src/99-appendices/06-meg-file-formats.md

It is RECOMMENDED that FIFF files with multiple parts use the `split-<index>`
entity to indicate each part.

> If there are split files and the optional `scans.tsv` is provided, remember to list all parts
> of the split file in `scans.tsv` and that the entries for the `acq_time` column in `scans.tsv`

Reviewer: ```suggestion
If there are split files and the optional `scans.tsv` is provided, remember to list the
files separately in `scans.tsv` and that the entries for the `acq_time` column in `scans.tsv`
```

I think it's confusing to refer to a "split file".

It is RECOMMENDED that FIFF files with multiple parts use the `split-<index>`
entity to indicate each part.

> Split files MUST be listed as separate recordings in the `scans.tsv` file.
> 

Reviewer 2: ```suggestion
If there are split files and the optional `scans.tsv` is provided, remember to list all parts
of the split file in `scans.tsv` and that the entries for the `acq_time` column in `scans.tsv`
MUST be identical, as described in
[Scans file](../03-modality-agnostic-files.md#scans-file).

```
